src/app/modules/recipe-list/components/recipe-carousel.component.ts

```diff
@@ -0,0 +1,29 @@
+import {Component, HostListener, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'app-recipe-carousel',
+  templateUrl: './recipe-carousel.component.html',
+  styleUrls: ['./recipe-carousel.component.scss']
+})
+export class RecipeCarouselComponent {
+
+  public limit = 3;
+  images = [
```

Comment:
images should goes from parent component, change to 
```suggestion
  @Input() images: Array<PleaseCreateProperTypeHere> = [];
```